tests/lldp/test_lldp.py

```diff
@@ -55,13 +56,13 @@ def test_lldp_neighbor(duthosts, enum_rand_one_per_hwsku_frontend_hostname, loca
         ])
 
     res = duthost.shell(
-        "docker exec -i lldp lldpcli show chassis | grep \"SysDescr:\" | sed -e 's/^\\s*SysDescr:\\s*//g'")
+        "docker exec -i lldp{} lldpcli show chassis | grep \"SysDescr:\" | sed -e 's/^\\s*SysDescr:\\s*//g'".format(
+            asic))
```

Comment:
format(asic)  -> format(asic if asic else'')
For Single asic, this should be empty string, not None

Reply:
Hi @mannytaheri , please help to get this PR merged. Suggest a change above to fix the lldp failure for single asic.

Reply:
done